INSTALL.rst

```diff
@@ -99,6 +99,31 @@ Now visit the following URL in your browser:
 
         $ docker-machine ip <machine-name>
 
+Docker Dev Environment installation
+-----------------------------------
+
+It is possible to use a docker development environment, for testing purposes. You can create and run it
+by following the next steps:
+
+- Build your ui by using the command `yarn build`, after installing the dependencies. The ``index.html``
+  file should be inside the ``ui/cap-react/dist folder``, in oreder to be mounted and used by the containers.
+- Use the command `docker-compose -f docker-compose.dev.yml up` to build and start the containers.
+  It is probably also necessary to open another shell and use `docker-compose -f docker-compose.full.yml run web-api sh scripts/init.sh`.
```

Comment:
* make command as codeblock `docker-compose -f docker-compose.dev.yml up` make command as codeblock. and add below the description for starting containers
* make `It is probably also necessary to open another shell and use "docker-compose -f docker-compose.full.yml run web-api sh scripts/init.sh".` another paragraph and with codeblock as above. Dont forget to change `.full` to `dev`